storage/simple-storage-client/src/lib.rs

```diff
@@ -29,7 +34,7 @@ impl SimpleStorageClient {
         }
     }
 
-    fn request<T: DeserializeOwned>(&self, input: StorageRequest) -> Result<T, Error> {
+    fn request<T: DeserializeOwned>(&self, input: StorageRequest) -> std::result::Result<T, Error> {
```

Comment:
what's wrong with using `anyhow::Result` here?

Reply:
hmm, `anyhow::Result` carries `anyhow::Error` which is not serializable.